### DISCUSSION TITLE
Shadow Window

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1007] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Certain stages (Mainly Mortal Kombat) have stuff like narrow floors, and having the shadow just display over the parts where the floor supposedly ends.

### Describe the solution you'd like

Have a window for shadows, so that they cut in a certain point.

**Comment by potsmugen** [comment, marked as answer]
Implemented by #2388.
